### PR TITLE
Fetch and display synced devices

### PR DESCRIPTION
**Pull request by G-Ray**

### CODE REVIEW

File: src/containers/Devices/Devices.jsx [hunk 1]
```diff
@@ -19,6 +19,10 @@ class Devices extends React.Component {
     this.handleAddDeviceClose = this.handleAddDeviceClose.bind(this)
   }
 
+  componentDidMount () {
+    this.props.fetchDevices()
+  }
+
   handleAddDeviceClick () {
     this.setState({ addDevice: true })
```
Comment (leventdem) [inline src/containers/Devices/Devices.jsx:27]: Info : if there is no signalhub, when I click on Add device button, there is no error message.  I only see the modal for a few seconds and it disappears.
Can you test it please ?

File: src/lib/masq.js [hunk 2]
```diff
@@ -115,6 +115,25 @@ class Masq {
...
+    let deviceName = `${capitalize(name)} - ${capitalize(os)}`
+
+    const count = devices.reduce((acc, { name }) => (
+      (name.substring(0, deviceName.length) === deviceName) ? acc + 1 : acc
+    ), 0)
+
+    if (count > 0) {
+      deviceName = `${deviceName} - ${count}`
+    }
+
```
Comment (leventdem) [inline src/lib/masq.js:130]: Ok, good solution in case there are the same device names.

**Review by leventdem — APPROVED** [review]
Ok, I only added a comment about the expected behaviour in case we want to add  a device but the signalhub is down.

File: src/containers/Devices/Devices.jsx (hunk 1 above)
Comment (G-Ray) [reply src/containers/Devices/Devices.jsx:27]: Good catch, I fixed the notification position

**Review by sujeebant — APPROVED** [review]

**Pull request merged**